core/runtime/TRTEngine.cpp

```diff
@@ -118,9 +118,11 @@ TRTEngine::TRTEngine(
       TORCHTRT_CHECK(
           (cuda_engine->getTensorIOMode(binding_name.c_str()) == nvinfer1::TensorIOMode::kINPUT),
           "Binding " << binding_name << " specified as input but found as output in TensorRT engine");
-      LOG_DEBUG("Input binding name: " << binding_name << "pyt arg idx: " << pyt_idx << ")");
+      LOG_DEBUG(
+          "Input binding name: " << binding_name << " has TensorRT binding index: " << trt_idx
+                                 << ", Torch binding index: " << pyt_idx);
       in_binding_map[trt_idx] = pyt_idx;
-      in_binding_names[pyt_idx] = _in_binding_names[pyt_idx];
+      in_binding_names[pyt_idx] = binding_name;
```

Comment:
Code clarity update - note that `binding_name = _in_binding_names[pyt_idx]` assigned above